test/views/ViewTestHelper.scala

```diff
@@ -37,6 +37,24 @@ trait ViewTestHelper extends SpecBase {
     view.title() mustBe s"${messages(titleMsgKey)} - ${messages("service.name")} - GOV.UK"
 
   def shouldContainBackLinkUrl(view: Document,
-                               url: String): Assertion =
-    view.html().contains(url) mustBe true
+                               url: String): Assertion = view.html().contains(url) mustBe true
+
+  def shouldContainTheElement(view: Document,
+                              id: Option[String] = None,
+                              classes: Option[String] = None): Assertion = {
+    val isElementIdPresent = id.fold(false)(id => view.select(s"#$id").size() >= 0)
+    val isElementClassPresent = classes.fold(false)(classes => view.select(s".$classes").size() >= 0)
+
+    assert(isElementIdPresent || isElementClassPresent)
+  }
+
+  def shouldNotContainTheElement(view: Document,
+                                 id: Option[String] = None,
+                                 classes: Option[String] = None): Assertion = {
+    val isElementIdPresent = id.fold(false)(id => view.select(s"#$id").size() == 0)
+    val isElementClassPresent = classes.fold(false)(classes => view.select(s".$classes").size() == 0)
+
+    assert(isElementIdPresent || isElementClassPresent)
```

Comment:
If your intention here is to check for both `ID` and `class`, you should use `&&` instead of `||`.

Current logic  will assert true if either an `ID` or `class` is present or absent, but not necessarily both.

Reply:
Yes, it has been done on purpose as either the element's id or classes can be passed as input and should result as true. Going forward, more specific methods will be added in ViewTestHelper that will check only based on Id, classes, attribute etc